.github/scripts/trymerge.py

```diff
@@ -1257,7 +1257,7 @@ def validate_revert(repo: GitRepo, pr: GitHubPR, *,
         commit_sha = commits[0]
     msg = repo.commit_message(commit_sha)
     rc = RE_DIFF_REV.search(msg)
-    if rc is not None and not can_skip_internal_checks:
+    if rc is not None and not skip_internal_checks:
         raise PostCommentError(f"Can't revert PR that was landed via phabricator as {rc.group(1)}")
```

Comment:
Nice! Can you please also update the comment error to explain the correct way to revert the PR? (Click the diff link in the PR body to go to the diff page and click the "unland" button on that page)